challenge_1/csharp/sven/src/Reverse.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Linq;
+
+namespace Challenge_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine(
+                args.Aggregate("", (sentence, word) => word.Aggregate("", (w, character) => character + w) + " " + sentence)
```

Comment:
Aggregate is a sequence accumulator. It is a linq method which applies a given function to each element of the sequence it operates on, while carrying the result forward.

I've used two Aggregate methods, one nested into the other. The effect is that the outer most Aggregate will swap the order of the words in a sequence (hello world! => world! hello).
The inner most Aggregate uses the same logic but operates on each word separately (hello => olleh)
The combination is that a given string is reversed.

Both aggregate methods I use takes 3 parameters.
I'll explain it for the outer most Aggregate:
- the seed (the empty string ""): is the initial value that will be present in the accumulator argument "sentence"
- the accumulator "sentence"
- the element to operate on "word"
The function to perform on each element is given after the "=>" operator as a lambda:
concatenate the (reverse of a) word to the front of the sentence.